clients/da-vinci-client/src/main/java/com/linkedin/davinci/StoreBackend.java

```diff
@@ -92,7 +90,6 @@ synchronized void delete() {
       version.delete();
     }
 
-    backend.getStoreRepository().unsubscribe(storeName);
```

Comment:
Why do we want to remove this?
Keeping the subscription only will let DaVinci application keep polling the backend to retrieve store metadata, which is not good for a deleted store.

Reply:
Yeah I think this makes sense. We can keep this one and only remove the above one, during the regular close.